_data-prepper/pipelines/configuration/sources/s3.md

```diff
@@ -45,24 +45,54 @@ In order to use the `s3` source, configure your AWS Identity and Access Manageme
 
 If your S3 objects or Amazon SQS queues do not use [AWS Key Management Service (AWS KMS)](https://aws.amazon.com/kms/), remove the `kms:Decrypt` permission.
 
+## Cross-account S3 access
+
+### Bucket ownership<a name="s3_bucket_ownership"></a>
+
+When Data Prepper gets data from the S3 bucket, it verifies the ownership of the bucket using the 
```

Comment:
```suggestion
When Data Prepper fetches data from an S3 bucket, it verifies the ownership of the bucket using the [bucket owner condition](https://docs.aws.amazon.com/AmazonS3/latest/userguide/bucket-owner-condition.html). By default, Data Prepper expects that an S3 bucket is owned by the same that owns the correlating SQS queue. When no SQS is provided, Data Prepper uses the ARN role in the `aws` configuration.
```

Reply:
I'm curious as the choice of "fetch" here. Amazon S3's API is called `GetObject`, so "get" seems closer to their API design. They also use the term "retrieve."

Reply:
I'm okay sticking with get then.